guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -12,6 +12,7 @@ If you lose connection to the shell where the installation command is running, s
 .Considerations
 
 * Use the `{installer-scenario} --help` command to display the available options and any default values.
+* For advanced installation options, use `{installer-scenario} --full-help`.
```

Comment:
```suggestion
* Use the `{installer-scenario} --full-help` command to display advanced options.
```

This is to keep the list item parallel to the one before, this helps users navigate the text more quickly.